source/index.ts

```diff
@@ -63,17 +62,23 @@ if (
 }
 
 class Chromium {
+  /**
+   * Determines the headless mode that chromium will run at
+   * https://developer.chrome.com/articles/new-headless/#try-out-the-new-headless
+   * @values true or "new"
+   */
+  static headlessMode: true | "new" = "new";
+
+  /**
+   * If true, the graphics stack and webgl is enabled,
+   * If false, webgl will be disabled. (If false, the swiftshader driver will also not extract)
+   */
+  static graphicsStackMode: boolean = true;
```

Comment:
My understanding is that in serverless environments we need to use swiftshader to use CPU only but we are now introducing a new configuration mode which would can disable webgl.

Would it make sense here to have the mode to have type of `"swiftshader" | "swiftshader-webgl" | "webgl-disabled"`?

the new option would be to use swiftshader to render webgl content, or ultimately disable webgl completely.

If the intention is to have an actual boolean maybe we can use something like `graphicsStackEnabled`?

Reply:
I think for most users, swiftshader can be disabled and not used at all, it saves at least a full second on my testing for initial extracts. Swiftshader/angle is what powers webgl, so I'm not sure what the difference would be for `swiftshader` or `swiftshader-webgl` (Maybe my comment was confusing about the swiftshader driver)